src/Build/Instance/TaskRegistry.cs

```diff
@@ -411,8 +412,8 @@ ElementLocation elementLocation
         )
         {
             TaskFactoryWrapper taskFactory = null;
-            bool retrievedFromCache;
-
+            bool retrievedFromCache = false;
```

Comment:
Why this change? You can inline it instead if you want with ...out bool retrievedFromCache);